### DISCUSSION TITLE
Can we discuss a port of your mod to fabric?

### DISCUSSION THREAD

**Discussion by AletropyS2** [P3pp3rF1y/SophisticatedBackpacks#711] [Q&A] [answered]
# Hello!

I've been making mods for minecraft for a while and I've always found fabric lighter and simpler, your backpacks mod is the best and most interesting mod you have among all versions and mod loaders. I wonder if I could port your mod to fabric, I don't want the credits or recognition, just that the large amount of players who opt for fabric can enjoy this mod, that's what's missing there.

_I love the mod and I make this request with great affection, I await your response_

**Comment by AletropyS2** [comment, marked as answer]
Edit.

I know the license you are using is GPL, so I could do without asking for it. But I would like to hear from you what you think of the mod for fabric

**Reply by P3pp3rF1y** [reply]
I am cool with you taking a stab at porting the mod, but one thing is certain here - this mod actually is pretty complex in terms of code / states and definitely requires to be ported by someone who is very decent with Java / mc dev so definitely not something to learn modded development on. Have you got experience with modded / Java?

**Reply by AletropyS2** [reply]
I've been working with mods for a few months, but I'm pretty experienced when it comes to object-oriented languages like Java or C#. I believe that with the logical knowledge I have I can port this to fabric. I'll take a look at the code and get back to you.

Thanks a lot for the feedback

**Reply by SmallYii** [reply]
May I ask, are you still working on fabric's Sophisticated Backpacks?

**Reply by ajaxnotmeta** [reply]
whelp here we goes my only hopes

**Reply by Democracyassassin** [reply]
@AletropyS2 are you any closer to porting sophisticated backpacks because i am making a server for me and my friends and this port would be the perfect backpack mod thx again

**Reply by Twisted-Code** [reply]
Given they don't have any public repositories mentioning "backpacks" I'm guessing it's not particularly close to release, if extant at all

**Reply by ManoYeet10** [reply]
Please do port this to Fabric, it may not seem like much but Sophisticated Backpacks is the best backpack mod in all of minecraft modding imo (Seriously, it's in essentially every modded let's play on youtube) but sadly it's not on fabric (and you know how the rest goes) (:
oh wait the author hasn't even replied to this thread
lol nvm then
(But someone plz port this I swear)

**Reply by superhulk117** [reply]
I don't even know how to make mods, and I want to learn just to port this over! :(

**Reply by PantherBrick** [reply]
Please do forge i am literally BEGGING YOU
